src/Components/CreateImageWizard/formComponents/ReviewStep.js

```diff
@@ -113,392 +63,106 @@ const ReviewStep = () => {
-                    >
-                      <Button
-                        data-testid="file-system-configuration-popover"
-                        variant="link"
-                        aria-label="File system configuration info"
-                        aria-describedby="file-system-configuration-info"
-                        className="pf-u-pt-0 pf-u-pb-0"
-                      >
-                        View partitions
-                      </Button>
-                    </Popover>
-                  </>
-                )}
-              </TextListItem>
-              {getState()?.values?.['file-system-config-radio'] ===
-                'manual' && (
-                <>
-                  <TextListItem component={TextListItemVariants.dt}>
-                    Image size (minimum)
-                    <Popover
-                      hasAutoWidth
-                      bodyContent={
-                        <TextContent>
-                          <Text>
-                            Image Builder may extend this size based on
-                            requirements, selected packages, and configurations.
-                          </Text>
-                        </TextContent>
-                      }
-                    >
-                      <Button
-                        variant="plain"
-                        aria-label="File system configuration info"
-                        aria-describedby="file-system-configuration-info"
-                        className="pf-c-form__group-label-help"
-                      >
-                        <HelpIcon />
-                      </Button>
-                    </Popover>
-                  </TextListItem>
-                  <TextListItem component={TextListItemVariants.dd}>
-                    {minSize}
-                  </TextListItem>
-                </>
-              )}
-            </TextList>
-            <Text component={TextVariants.h3}>Additional packages</Text>
-            <TextList component={TextListVariants.dl}>
-              <TextListItem component={TextListItemVariants.dt}>
-                Chosen
-              </TextListItem>
-              <TextListItem
-                component={TextListItemVariants.dd}
-                data-testid="chosen-packages-count"
-              >
-                {getState()?.values?.['selected-packages']?.length || 0}
-              </TextListItem>
-            </TextList>
+            <Text component={TextVariants.h3}>
+              Bare metal - Installer (.iso)
+            </Text>
+            <TargetEnvOtherList />
           </TextContent>
-        </Tab>
-      </Tabs>
+        )}
+      </ExpandableSection>
+      <ExpandableSection
+        toggleContent={'File system configuration'}
+        onToggle={onToggle}
+        isExpanded={isExpanded['file-system-configuration']}
+        isIndented
+        data-testid="file-system-configuration-expandable"
+      >
+        <FSCList minSize={minSize} />
```

Comment:
Mentioned in another comment above but we could just manage minSize inside of the `<FSCList />` component itself couldn't we?